### ISSUE TITLE
I can't receive mail

### ISSUE THREAD

**Issue by zhangiicc** [naver/pinpoint-docker#18]
cat .env
```
BATCH_ENABLE=true
BATCH_SERVER_IP=127.0.0.127
BATCH_FLINK_SERVER=pinpoint-flink-jobmanager

CONFIG_SHOW_APPLICATIONSTAT=true

JDBC_DRIVER=com.mysql.jdbc.Driver
JDBC_URL=jdbc:mysql://pinpoint-mysql:3306/pinpoint?characterEncoding=UTF-8
JDBC_USERNAME=admin
JDBC_PASSWORD=admin
MAIL_HOST=stmp.***.com
MAIL_PORT=25
MAIL_USERNAME=jie1.zhang@****.com
MAIL_PASSWORD=****
MAIL_PROPERTIES_MAIL_TRANSPORT_PROTOCOL=smtp
MAIL_PROPERTIES_MAIL_SMTP_AUTH=true
MAIL_PROPERTIES_MAIL_SMTP_PORT=25
MAIL_PROPERTIES_MAIL_SMTP_FROM=jie1.zhang@****.com
MAIL_PROPERTIES_MAIL_STARTTLS_ENABLE=true
MAIL_PROPERTIES_MAIL_STARTTLS_REQUIRED=true
MAIL_PROPERTIES_MAIL_DEBUG=true
```

**Comment by zhangiicc** [comment]
Is it less configuration? 
cat  applicationContext-batch.xml 
```
PINPOINT_VERSION=1.8.2

### Pinpoint-Hbase

PINPOINT_HBASE_NAME=pinpoint-hbase
#config for hbase in external docker
EXTERNAL_HBASE_PORT=2180

### Pinpoint-mysql
MYSQL_ROOT_PASSWORD=root123
MYSQL_USER=admin
MYSQL_PASSWORD=admin
MYSQL_DATABASE=pinpoint

### Pinpoint-Web

PINPOINT_WEB_NAME=pinpoint-web
EXTERNAL_HBASE_PORT=2180

### Pinpoint-mysql
MYSQL_ROOT_PASSWORD=root123
MYSQL_USER=admin
MYSQL_PASSWORD=admin
MYSQL_DATABASE=pinpoint

### Pinpoint-Web

PINPOINT_WEB_NAME=pinpoint-web

WEB_PAGE_PORT=8079

CLUSTER_ENABLE=true
#CLUSTER_WEB_TCP_PORT=9997
CLUSTER_ZOOKEEPER_ADDRESS=zoo1

ADMIN_PASSWORD=admin

ANALYTICS=true

HBASE_HOST=pinpoint-hbase
HBASE_PORT=2181

WEB_DEBUG_LEVEL=INFO

BATCH_ENABLE=true
BATCH_SERVER_IP=127.0.0.127
BATCH_FLINK_SERVER=pinpoint-flink-jobmanager

CONFIG_SHOW_APPLICATIONSTAT=true

JDBC_DRIVER=com.mysql.jdbc.Driver
JDBC_URL=jdbc:mysql://pinpoint-mysql:3306/pinpoint?characterEncoding=UTF-8
JDBC_USERNAME=admin
JDBC_PASSWORD=admin
MAIL_HOST=stmp.zkh360.com
MAIL_PORT=25
MAIL_USERNAME=jie1.zhang@zkh360.com
MAIL_PASSWORD=201253Zhao
MAIL_PROPERTIES_MAIL_TRANSPORT_PROTOCOL=smtp
MAIL_PROPERTIES_MAIL_SMTP_AUTH=true
MAIL_PROPERTIES_MAIL_SMTP_PORT=25
MAIL_PROPERTIES_MAIL_SMTP_FROM=jie1.zhang@zkh360.com
MAIL_PROPERTIES_MAIL_STARTTLS_ENABLE=true
MAIL_PROPERTIES_MAIL_STARTTLS_REQUIRED=true
MAIL_PROPERTIES_MAIL_DEBUG=true

```

**Comment by RoySRose** [comment]
Hello, @zhangiicc 
for Pinpoint v1.8.2 please check the [alarm guide](https://naver.github.io/pinpoint/1.8.2/alarm.html)
So far I know, from v1.9.0-Snapshot default(basic) alarm is implemented. Until then, You need to implement some code in Pinpoint-Web

**Comment by zhangiicc** [comment]
v1.9.0 ?

**Comment by zhangiicc** [comment]
The latest version is 1.8.2

**Comment by RoySRose** [comment]
Yes. Latest released version is 1.8.2.
Latest snapshot is 1.9.0. You can either wait for the release of 1.9.0. 
Or
Build current snapshot for yourself.
